### PR TITLE
teufelssecurity: add location

### PR DESCRIPTION
**Pull request by noxilixon**
Mac overwrite for mesh ap needs to be added. I don't have access to a device right now.

### CODE REVIEW

File: locations/teufelssecurity.yml [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    role: ap
+    model: mikrotik_sxtsq-5-ac
+    wireless_profile: freifunk_default
+
+networks:
+
+  - vid: 20
+    role: mesh
+    name: mesh_ap
+    prefix: 10.31.243.64/32
```
Comment (FFHener) [inline locations/teufelssecurity.yml:33]: This Prefix needs to be part of the Meshsubnet

**Review by FFHener — CHANGES_REQUESTED** [review]

**Review by FFHener — APPROVED** [review]
lgtm

**Comment by FFHener** [comment]
Will keep this as a draft until we builded this locations as there might be changes again

**Comment by noxilixon** [comment]
I updated the coordinates. Were there other changes during the installation @FFHener ? If not we can merge this.

File: locations/teufelsberg.yml [hunk 2]
```diff
@@ -204,7 +204,7 @@ location__channel_assignments_11a_standard__to_merge:
   # all APs on channel 36 for now, to allow meshing for standard Falter APs
   # TODO: Change this once the installation is more optimized
   teufelsberg-core: 40-20
-  teufelsberg-ap1: 36-20
+  teufelsberg-ap1: 44-20
```
[the remark this answers is not in the document]
Comment (noxilixon) [reply locations/teufelsberg.yml:207]: Removed it, not sure why it existed in the first place.

File: locations/teufelsberg.yml [hunk 3]
```diff
@@ -2,8 +2,8 @@
 
 location: teufelsberg
 location_nice: Teufelsberg
-latitude: 52.49800
-longitude: 13.24052
+latitude: 52.49649
+longitude: 13.23970
```
Comment (Noki) [inline locations/teufelsberg.yml:6]: Don't change this. It's the core location at the tower.

**Pull request merged**